src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -211,11 +212,11 @@ public class MyClass {
         objects.
       </p>
       <p>
-        Main Java grammar skips any whitespaces and newlines, so in Java Abstract Syntax Tree there
-        are no whitespace/newline nodes. In Javadoc comment every whitespace matters, and Javadoc
-        Checks need all those whitespaces and newline nodes to verify format and content of the
-        Javadoc comment. Because of that Javadoc grammar includes all whitespaces, newlines to
-        the parse tree
+        The main Java grammar skips any whitespaces and newlines, so in Java Abstract Syntax Trees
+        there are no whitespace/newline nodes. In Javadoc comments every whitespace matters, and
```

Comment:
, so in Java Abstract Syntax Trees there are no whitespace/newline nodes.

`, so there are no whitespace/newline nodes in Java Abstract Syntax Tree.`